src/defaultsettings.cpp

```diff
@@ -57,6 +57,10 @@ void set_default_settings(Settings *settings)
 	settings->setDefault("curl_verify_cert", "true");
 	settings->setDefault("enable_remote_media_server", "true");
 	settings->setDefault("enable_client_modding", "false");
+	settings->setDefault("enable_knockback", "true");
+	settings->setDefault("knockback_default_duration", "0.3");
```

Comment:
Hmm because movement is clientside, so here you are creating an instantly-visible knockback which is an illusion of the other player being knocked, to avoid network lag causing a delayed visible knockback caused by the actual movement of the other player?
So you are blocking the position of the player receiving the knockback to not interfere with the clientside knockback animation?
So yes seems needed, sorry i misunderstood, i thought this was the same issue as disabling the controls for the player that is knocked-back, which doesn't seem necessary.